src/client/playwright.config.ts

```diff
@@ -34,7 +34,18 @@ export default defineConfig({
   projects: [
     {
       name: 'chromium',
-      use: { ...devices['Desktop Chrome'] },
+      use: {
+        ...devices['Desktop Chrome'],
+        // TODO: Remove all 3 of these security bypasses.
+        // These are not necessary locally, but in the pipeline the ID server doesn't have a valid cert
+        // This causes a the following error when the login button is clicked during a Playwright test
+        // Error: page.waitForURL: net::ERR_CERT_AUTHORITY_INVALID
+        bypassCSP: true,
+        launchOptions: {
+          args: ['--disable-web-security', '--allow-insecure-localhost'],
+        },
+        ignoreHTTPSErrors: true,
+      },
```

Comment:
This change is NOT staying for long. Basically since updating the mock oidc server image, the https certificate isn't correctly being loaded by the pipeline. It works perfectly fine locally but for some reason I haven't yet found, the id server isn't being trusted in the pipeline. I'm running out of time and Product's patience for this particular ticket so will tackle this in another PR.